pkgs/development/python-modules/theano-pymc/default.nix

```diff
@@ -0,0 +1,55 @@
+{ lib
+, fetchPypi
+, buildPythonPackage
+, pythonOlder
+, pandas
+, numpy
+, scipy
+, filelock
+, pytest
+, nose
+, parameterized
+}:
+
+buildPythonPackage rec {
+  pname = "Theano-PyMC";
+  version = "1.1.2";
+  disabled = pythonOlder "3.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "5da6c2242ea72a991c8446d7fe7d35189ea346ef7d024c890397011114bf10fc";
+  };
+
+  # No need for coverage stats in Nix builds
+  postPatch = ''
+    substituteInPlace setup.py --replace ", 'pytest-cov'" ""
+  '';
+
+  propagatedBuildInputs = [
+    pandas
+    numpy
+    scipy
+    filelock
+  ];
+
+  checkInputs = [
+    pytest
+    nose
+    parameterized
+  ];
```

Comment:
If you don't run the test suite, you likely won't need the `checkInputs` either, I guess. But maybe there is a point in keeping them in order to run a part of the tests manually. I don't know what the recommendation here is.

Reply:
I don't really know if I should remove them either, many other packages don't run tests but, still have the `CheckInputs`. But I don't know if its's an artifact or what should be done

Reply:
If they never worked we don't need checkInputs. If they break at some point they should be kept.